src/main/java/seedu/duke/data/notebook/Notebook.java

```diff
@@ -52,4 +52,8 @@ public void addNote(Note note) throws SystemException {
     public void deleteNote(int note) throws SystemException {
```

Comment:
Is the throws SystemException necessary here? I don't see it being thrown.
```suggestion
    public void deleteNote(int note) {
```

Reply:
I actually didnt change that but yea it doesn't throw an exception